src/main/webapp/WEB-INF/fenixedu-academic/accounting/heading-event.jsp

```diff
@@ -1,7 +1,15 @@
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions"%>
 <%@ page trimDirectiveWhitespaces="true" %>
 
-<h2>
+<c:if test="${fn:containsIgnoreCase(requestScope['javax.servlet.forward.request_uri'], 'owner-accounting-events')}">
+    <a href="<%= request.getContextPath() %>/owner-accounting-events/${event.externalId}/details">
+        ${event.description}
+    </a>
+</c:if>
+<c:if test="${not fn:containsIgnoreCase(requestScope['javax.servlet.forward.request_uri'], 'owner-accounting-events')}">
     <a href="<%= request.getContextPath() %>/accounting-management/${event.externalId}/details">
         ${event.description}
     </a>
-</h2>
+</c:if>
```

Comment:
This is a little bit messy. Maybe there is another way to accomplish this? Each controller should add their eventDetailsUrl to the model and this jsp should use it.